src/material/tabs/tab-group.ts

```diff
@@ -243,11 +248,26 @@ export abstract class _MatTabGroupBase extends _MatTabGroupMixinBase implements
         }
       }
 
-      this._subscribeToTabLabels();
```

Comment:
I remove this call because I noticed that there's another one just like it a few lines above. It seems like an issue when resolving merge conflicts.